corehq/apps/cleanup/management/commands/populate_sql_model_from_couch_model.py

```diff
@@ -108,37 +106,14 @@ def migrate_from_migration(cls, apps, schema_editor):
 
         command_name = cls.__module__.split('.')[-1]
         if to_migrate < cls.AUTO_MIGRATE_ITEMS_LIMIT:
-            try:
-                call_command(command_name)
-                remaining = cls.count_items_to_be_migrated()
-                if remaining != 0:
-                    migrated = False
-                    print(f"Automatic migration failed, {remaining} items remain to migrate.")
-                else:
-                    migrated = True
-            except Exception:
-                traceback.print_exc()
+            safely_run_management_command(command_name, required_commit=cls.commit_adding_migration())
+            remaining = cls.count_items_to_be_migrated()
+            if remaining != 0:
+                print(f"Automatic migration failed, {remaining} items remain to migrate.")
```

Comment:
Previously the `migrated` flag was set to false, ultimately resulting in `sys.exit(1)`, if there were remaining items, but now that doesn't appear to be the case. Can you explain why this is still safe?

It looks like `safely_run_management_command` will call `sys.exit(1)` if there is an error, but that happens before this message is printed. Is this even still necessary? I would not expect anyone to ever notice this in a stream of migration messages unless it happened to be the last one that was run.

Looks similar edits have been done in other migrations below as well. I didn't comment on those, but the same applies.

Reply:
Thanks good point. I think this would just require adding a `sys.exit(1)` after the message is printed

Reply:
d42d0ffd009